examples/advanced/ads.js

```diff
@@ -18,6 +18,13 @@ var Ads = function() {
 
   this.player = videojs('content_video');
 
+  this.adTagBox = document.getElementById('tagText');
```

Comment:
adTagInput is a little more clear, and might as well change "tagText" id to tagInput to match.